src/res/strings_utf8.rc

```diff
@@ -272,7 +272,7 @@ BEGIN
 END
 
 // by Ben Tsui
-LANGUAGE LANG_CHINESE_SIMPLIFIED, SUBLANG_CHINESE_SIMPLIFIED
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
```

Comment:
we added these tho
https://github.com/mingw-w64/mingw-w64/blame/0f94270b0007d892b4cd532d9a8a63db24b52ac0/mingw-w64-headers/include/winnt.rh#L39

Reply:
`LANGUAGE LANG_CHINESE_TRADITIONAL, SUBLANG_CHINESE_TRADITIONAL` breaks LLVM's llvm-rc with  `llvm-rc: Primary language ID (31748) does not fit in 10 bits.`

I just checked with GNU windres and MSVC resource compiler and confirmed it is actually valid. I guess it is a LLVM bug then.

`LANG_CHINESE_SIMPLIFIED` is fine (which is equal to `LANG_CHINESE` anyway).

Reply:
thank you